nixos/modules/services/monitoring/scrutiny-collector.nix

```
@@ -0,0 +1,123 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+
+  prettyJSON = conf:
+    pkgs.runCommand "loki-config.json" { } ''
+      echo '${builtins.toJSON conf}' | ${pkgs.jq}/bin/jq 'del(._module)' > $out
+    '';
+
+in
+{
+  options.services.scrutiny-collector = {
+    enable = mkEnableOption "scrutiny-collector";
+
+    package = mkPackageOption pkgs "scrutiny-collector" { };
+
+    api-endpoint = mkOption {
+      type = types.str;
+      default = "http://localhost:8080";
+      description = mdDoc ''
+        The api server endpoint
+      '';
+    };
+
+    log-file = mkOption {
+      type = types.nullOr types.path;
+      default = null;
+      description = mdDoc ''
+        Path to file for logging. Leave empty to use STDOUT
+      '';
+    };
+
+    debug = mkEnableOption "Enable debug logging";
```

`mkEnableOption` can be a bit surprising - it automatically prepends *Whether to enable* before the string. This would then become "Whether to enable Enable debug logging".

```suggestion
    debug = mkEnableOption "debug logging";
```